src/test_ops.rs

```diff
@@ -750,9 +950,24 @@ fn parse_atom(a: &mut Allocator, v: &str) -> NodePtr {
 }
 
 fn pop_token<'a>(s: &'a str) -> (&'a str, &'a str) {
-    match s.split_once(" ") {
-        Some((first, rest)) => (first.trim(), rest.trim()),
-        None => (s.trim(), ""),
+    if s.starts_with("\"") {
```

Comment:
this was made a little bit more sophisticated to support quoted atoms with spaces inside the quotes.